spark/dl/src/main/scala/com/intel/analytics/bigdl/example/treeLSTMSentiment/fetch_and_preprocess.py

```diff
@@ -0,0 +1,392 @@
+"""
```

Comment:
Please add a `fileSet` to `https://github.com/intel-analytics/BigDL/blob/master/spark/dist/assembly/dist.xml`, to copy this python file to dist/bin